src/OracleService/Protocols/OracleNeoFSProtocol.cs

```diff
@@ -0,0 +1,126 @@
+using Google.Protobuf;
+using Neo.Cryptography.ECC;
+using Neo.Network.P2P.Payloads;
+using Neo.Wallets;
+using NeoFS.API.v2.Client;
+using NeoFS.API.v2.Client.ObjectParams;
+using NeoFS.API.v2.Cryptography;
+using NeoFS.API.v2.Refs;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Object = NeoFS.API.v2.Object.Object;
+using Range = NeoFS.API.v2.Object.Range;
+
+namespace Neo.Plugins
+{
+    class OracleNeoFSProtocol : IOracleProtocol
+    {
+        private readonly byte[] privateKey;
+
+        public void Configure()
+        {
+        }
+
+        public OracleNeoFSProtocol(Wallet wallet, ECPoint[] oracles)
+        {
+            privateKey = oracles.Select(p => wallet.GetAccount(p)).Where(p => p is not null && p.HasKey && !p.Lock).FirstOrDefault()?.GetKey().PrivateKey;
```

Comment:
What if the wallet doesn't contain the oracle private key?

Reply:
It will throw a exception. 
Because privateKey will be null.
`Client client = new Client(privateKey.LoadPrivateKey(), host, 120000);`